src/lexer/comments.ts

```diff
@@ -67,7 +67,6 @@ export function skipMultiLineComment(parser: ParserState, state: LexerState): Le
   const { index } = parser;
   while (parser.index < parser.end) {
     while (parser.currentChar === Chars.Asterisk) {
-      state &= ~LexerState.LastIsCR;
```

Comment:
This entire single line comment can be rewritten and optimized for this purpose I guess. Look at this pseudo code

```ts
let lastIsCR = 0

while (this.hasNext()) {
    switch (this.current()) {
    case "\r": lastIsCR = 2 // So it gets decremented to 1
    case "\n": case "\u2028": case "\u2029":
        if (!--lastIsCR) this.line++
        this.column = 0
        this.index++
        this.hasLineTerminator = true
        return // end loop
    default:
        if (lastIsCR) { this.line++; lastIsCR = 0 }
        this.column++
        this.index++
    }
}
```

Reply:
This is no need for specific handling of CR/LF anywhere other than `advanceChar`, and there it need only treat `CR`, `CRLF`, `LFCR` as a line `LF`. Similarly, the only place column is reset, and line incremented should be in `consumeLineFeed`.

Reply:
Incidentally, I couldn't find a constant for the `8` and `8232` bit-flags used `comments.ts`.